lesson_1/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { config } from 'dotenv';
+import { route } from './router/index.ts';
+
+config();
+
+console.log('📋 Habit Tracker CLI');
+console.log('Enter commands (press Ctrl+C to exit)\n');
+
+process.stdin.setEncoding('utf8');
+process.stdout.write('> ');
+
+process.stdin.on('data', (data: string) => {
+  const args = data.trim().split(' ');
+  if (args[0]) {
+    route(args);
```

Comment:
Due to comment about I/O

```suggestion
   console.log(route(args));
```